site/content/en/docs/getting_started/overview.md

```diff
@@ -0,0 +1,209 @@
+---
+title: 'CVAT Overview'
+linkTitle: 'Introduction to CVAT'
+weight: 1
+description: 'CVAT: The open-source tool for image and video annotation and data labeling'
+---
+
+Machine learning systems often struggle due to
```

Comment:
Please fix formatting across the whole document. Keep the line length is ~80 characters. You can use some tools to fix that.